### ISSUE TITLE
Type of onValidateMerchant function is wrong

### ISSUE THREAD

**Issue by tlenclos** [Adyen/adyen-web#1498]
**Describe the bug**
Either `onValidateMerchant` function type is wrong or implementation.

Third argument is not an event but the validationURL.

**To Reproduce**
Configure Apple pay like the types suggest, third param is a string instead of an event.

```ts
applepay: {
    amount: {
        value: order.totalPrice!,
        currency: site.defaultCurrency?.code!,
    },
    countryCode: country?.countryISO,
    onSubmit: onPaymentSubmit,
    onValidateMerchant: (resolve, reject, event) => {
        customCall(event.validationURL)
    },
```

**Expected behavior**
Type should be 

```ts
    onValidateMerchant?: (resolve, reject, validationURL: string) => void;
```

https://github.com/Adyen/adyen-web/blob/3db67ba4e040f794a27b555a737b1b4d828c611e/packages/lib/src/components/ApplePay/types.ts#L132

**Screenshots**

![image](https://user-images.githubusercontent.com/686101/154113702-135e0f1d-6291-4a9b-8df9-c21b2016f301.png)

**Comment by ribeiroguilherme** [comment]
Hi @tlenclos . Thanks for reporting it. I will update this ticket once the fix is released. Cheers!

**Comment by ribeiroguilherme** [comment]
Hi @tlenclos . We released 5.9.0 which contains the fix for this issue. Thanks for reaching out!
